# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/html/HTMLIFrameElement.cpp
# Commit: de69e3c1635a
# Full Hash: de69e3c1635a311b5b22716b65b93fc1899a37a4
# Author: Andreas Farre <farre@mozilla.com>
# Date: 2020-08-01 09:48:10
# Regressor Bug: 1613431
# File Overlap Count: 1
# Description:
#   Bug 1613431 - Part 3: Handle synced setters return value. r=nika
#   
#   Depends on D83646
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D83647
# ==============================================================================

diff -r b326d33f98cb -r de69e3c1635a dom/html/HTMLIFrameElement.cpp
--- a/dom/html/HTMLIFrameElement.cpp	Fri Jul 31 13:37:00 2020 +0000
+++ b/dom/html/HTMLIFrameElement.cpp	Fri Jul 31 13:37:13 2020 +0000
@@ -187,8 +187,8 @@
                aName == nsGkAtoms::mozallowfullscreen) {
       if (mFrameLoader) {
         if (auto* bc = mFrameLoader->GetExtantBrowsingContext()) {
-          // This can go away once we remove the featurePolicy pref.
-          bc->SetFullscreenAllowedByOwner(AllowFullscreen());
+          MOZ_ALWAYS_SUCCEEDS(
+              bc->SetFullscreenAllowedByOwner(AllowFullscreen()));
         }
       }
     }